vMenu/menus/VehicleOptions.cs

```diff
@@ -1943,7 +1943,8 @@ public void UpdateMods(int selectedIndex = 0)
                     "Benny's (1)",  // 8
                     "Benny's (2)",  // 9
                     "Open Wheel",   // 10
-                    "Street"        // 11
+                    "Street",       // 11
+                    "Track"         // 12    
                 };
                 var vehicleWheelType = new MenuListItem("Wheel Type", wheelTypes, MathUtil.Clamp(GetVehicleWheelType(veh.Handle), 0, 11), $"Choose a ~y~wheel type~s~ for your vehicle.");

```

Comment:
Make sure to also change the number here to 12, since there's 12 options

Reply:
Done